pkg/virt-operator/resource/generate/install/generated_mock_strategy.go

```diff
@@ -9,7 +9,7 @@ import (
 	gomock "github.com/golang/mock/gomock"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	types "k8s.io/apimachinery/pkg/types"
-	v1beta1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1"
+	v10 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
```

Comment:
You used `apiregv1` in other places, any specific reason to use something else here?

Reply:
That's auto-generated by `make generate`, I don't think I can change the name somehow.

Reply:
Ahh, yes, this is a generated file. :/